src/classes/classes-public.ts

```diff
@@ -0,0 +1,83 @@
+//public classes
+
+/*
+
+Classes are a common abstraction used in object-oriented programming (OOP) languages to describe data structures known as objects.​
```

Comment:
_We should decrease the amount of text going in the presentation:_ 

In object-oriented programming (OOP), classes serve as blueprints for creating objects. A class declaration is made using the 'class' keyword, and instances are created using the 'new' keyword followed by the class name. 

In TypeScript classes have visibility modifiers:
- public (accessible outside the class)
- protected (accessible by instances and subclasses)
- private (inaccessible outside the class).